### DISCUSSION TITLE
Is there a way to see which message is missing upon the 'MISSING_MESSAGE' error?

### DISCUSSION THREAD

**Discussion by wscourge** [amannn/next-intl#704] [Q&A] [answered]
See the title. I'd like to see what was passed to `t()` when the error is thrown. Ideally with a namespace, if there was any.

This is what I'm seeing right now during `next build`:

```
h [Error]: MISSING_MESSAGE
    at t.messages (/home/.next/server/chunks/1991.js:5:54364)
    at t.createBaseTranslator (/home/.next/server/chunks/1991.js:5:54414)
    at /home/.next/server/chunks/1991.js:5:52519
    at t.eX (/home/.next/server/chunks/1991.js:5:52602)
    at /home/.next/server/chunks/1991.js:5:44609
    at runNextTicks (node:internal/process/task_queues:60:5)
    at listOnTimeout (node:internal/timers:540:9)
    at process.processTimers (node:internal/timers:514:7)
    at async p (/home/.next/server/chunks/7837.js:4:15988) {
  code: 'MISSING_MESSAGE',
  originalMessage: undefined
}
```

**Comment by amannn** [comment, marked as answer]
I think you’re looking for this: https://next-intl-docs.vercel.app/docs/usage/configuration#error-handling

The error message you’re seeing is intentionally pretty sparse due to minification in production builds. If you trigger the error in development, you’ll get more details by default.

This makes me wonder though if we could include at least the namespace and key by default. This is probably the most frequent error that’s triggered by next-intl and including these details should be possible without any significant change in regard to the production bundle size.

**Reply by wscourge** [reply]
Thanks, that's it! I definitely think that including namespace and key by default would be helpful.

**Reply by amannn** [reply]
This has been addressed in `next-intl@3.3.1`—you'll now see the full namespace and key when a missing message is encountered.

Thanks for bringing this up @wscourge!

**Reply by wscourge** [reply]
Thank you!
